lib/iiif_print/metadata.rb

```diff
@@ -67,7 +67,7 @@ def scrub(value)
     def cast_to_value(field_name:, options:)
       if options&.[](:render_as) == :faceted
         faceted_values_for(field_name: field_name)
-      elsif options&.[](:render_as) == :rights_statement || options&.[](:render_as) == :license
+      elsif qa_field?(field_name: options&.dig(:render_as)) || qa_field?(field_name: field_name)
```

Comment:
I'm not following what's happening here.  Also, we're calling `qa_field?` twice.  I suspect the following is more appropriate:

```ruby
elsif qa_field?(field_name: options&.dig(:render_as) || field_name)
```